packages/editor/src/components/convert-to-group-buttons/convert-button.js

```diff
@@ -32,7 +32,7 @@ export function ConvertToGroupButton( {
 					icon={ Group }
 					onClick={ onConvertToGroup }
 				>
-					{ _x( 'Group', 'verb' ) }
+					{ _x( 'Create a Group Block', 'Creates a new Group block containing the selected blocks in the Editor' ) }
```

Comment:
The only concern I have is that it is implying you are just creating a new Block. It doesn't imply that you are necessarily containing the selected blocks _within_ a new group block.

Reply:
Yeah, I struggled a bit with this one. "Group Blocks" didn't actually make a ton of sense, since you can trigger this from just one block. Maybe "Add to a Group Block"?